frontend/src/scenes/App.tsx

```diff
@@ -17,6 +17,7 @@ import { BackTo } from 'lib/components/BackTo'
 import { Papercups } from 'lib/components/Papercups'
 import { appLogicType } from './AppType'
 import { PreflightStatus } from '~/types'
+import { models } from '~/models'
```

Comment:
Nitpicking: mixing relative `./AppType`, tilde absolute `~/models` and absolute `lib/components/Papercups` imports is confusing.

Reply:
Agreed, though I'd like to tackle this another time. Absolute paths for `lib` and `scenes` work without the tilde, everything else requires a tilde. That's confusing and touches a lot of files.